sdk/documenttranslation/azure-ai-documenttranslation/tests/test_cancel_job.py

```diff
@@ -15,9 +16,9 @@ class TestCancelJob(DocumentTranslationTest):
 
     @DocumentTranslationPreparer()
     @DocumentTranslationClientPreparer()
-    def test_list_statuses(self, client):
+    def test_cancel_job(self, client):
         # prepare containers and test data
-        blob_data = [b'This is some text']
+        blob_data = [Document(data=b'This is some text')]
```

Comment:
not related to your PR but how do u guarantee that the job is still running before you call cancel?

Reply:
Good point. Looking at the recordings, this was in a "NotStarted" state before it got canceled. Might make sense to add a few extra documents into the input to make sure this doesn't finish too quick